### DISCUSSION TITLE
When will milvus v2.0 support GPU

### DISCUSSION THREAD

**Discussion by xiaojuan8** [milvus-io/milvus#15564] [Q&A and General discussion]
Hi, milvus support GPU on v1.0, when will GPU is supported on v2.0? Thanks.

**Comment by yhmo** [comment]
In the previous plan, the gpu support is in v2.1. But now it is excluded from v2.1 scope:
https://wiki.lfaidata.foundation/pages/viewpage.action?pageId=52267944

The reason is we feel it is a challenge/risk, we want to ensure priority for the "string type field".

**Reply by xiaojuan8** [reply]
> The reason is we feel it is a challenge/risk, we want to ensure priority for the "string type field".

so, Is it possible for v2.0 to support GPU mode? When will GPU mode be released?

**Reply by yhmo** [reply]
Yes, the v2.x definitely can support GPU mode. The v2.1 is planned to be released on April 30, 2022. I suppose GPU mode is available on v2.2.

**Reply by xiaofan-luan** [reply]
Due to lack of man power, we delayed GPU support,  but soon we will have GPU support
